doc - menubar enhancement (#59)
NativePHP/nativephp.com: resources/views/docs/1/the-basics/menu-bar.md

    ->icon(storage_path('app/menuBarIconTemplate.png'));
```

### Vibrancy and Background Color

> For macOS, use the `HasVibrancy` trait to apply window vibrancy effects:

Reviewer: ```suggestion
For macOS, you may use the `vibrancy` method to apply window vibrancy effects:
```

    ->height(600);
```

### Resizable Window

Allow or prevent resizing of the menu bar window:
```php
MenuBar::resizable(false);
```

### Positioning

> Set the position of the menu bar window using the `HasPositioner` trait:

Reviewer: ```suggestion
You may manually set the position of the menu bar window:
```

To learn more about NativePHP's broadcasting capabilities, please refer to the [Broadcasting](/docs/digging-deeper/broadcasting) section.

### Listening for Custom Events

> Attach a custom event handler to the menu bar:

Reviewer: ```suggestion
Attach a custom event that should be fired when the menu bar icon is clicked:
```

```php

> MenuBar::create()->event('custom-event-name');

Reviewer: ```suggestion
MenuBar::create()->event(MenuBarClicked::class);
```
